pkg/controller/importconfig/cluster_info.go

```diff
@@ -32,14 +32,57 @@ import (
 	"k8s.io/apiserver/pkg/storage/names"
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/tools/clientcmd"
-	clientcmdapi "k8s.io/client-go/tools/clientcmd/api"
-	clientcmdlatest "k8s.io/client-go/tools/clientcmd/api/latest"
 	"k8s.io/utils/pointer"
 
 	authv1 "k8s.io/api/authentication/v1"
+	clientcmdapi "k8s.io/client-go/tools/clientcmd/api"
+	clientcmdlatest "k8s.io/client-go/tools/clientcmd/api/latest"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+//go:embed manifests
+var manifestFiles embed.FS
+
+const bootstrapSASuffix = "bootstrap-sa"
+
+/* #nosec */
+const (
+	registrationOperatorImageEnvVarName = "REGISTRATION_OPERATOR_IMAGE"
+	registrationImageEnvVarName         = "REGISTRATION_IMAGE"
+	workImageEnvVarName                 = "WORK_IMAGE"
+	defaultImagePullSecretEnvVarName    = "DEFAULT_IMAGE_PULL_SECRET"
+)
+
```

Comment:
I think this and render should be in a separated package. And you just need to provide a func to return crds and manifests from that package.

Reply:
Agree, I planned to spilt this part in a follow-up PR to wait for recent manifests changes to get merged, but we can also split it first.

An independent pkg was created.